server/auth/auth_test.go

```diff
@@ -931,6 +931,7 @@ func TestAccountErrors(t *testing.T) {
 
 func TestRoute(t *testing.T) {
 	user := tNewUser(t)
+	rig.signer.sig = user.randomSignature()
 	connectUser(t, user)
```

Comment:
All these panicked in `connectUser` when given a nil sig.  Each one was relying on it being set in a prior test.